plugins/hls-haddock-comments-plugin/test/testdata/test/testdata/QualFunction.expected.hs

```diff
@@ -0,0 +1,6 @@
+module QualFunction where
+
+f :: (Show a, Show b) => a -- ^ 
```

Comment:
Did you mean to include this?

Reply:
I guess the author of `hls-haddock-comments-plugin` forgot to commit this file... And the CI will never discover it, because the test will simply pass if there is no golden file. It is generated because I have run all tests locally.
As the test passes now, including this won't hurt, I think?

Reply:
Oh I see, it's an expected file. Then yes, check it in!